apps/web/nuxt.config.ts

```diff
@@ -68,6 +68,10 @@ export default defineNuxtConfig({
       showConfigurationDrawer: process.env.SHOW_CONFIGURATION_DRAWER === '1',
       defaultItemsPerPage: Number(process.env.DEFAULT_FEEDBACK_ITEMS_PER_PAGE ?? 10),
       headerLogo: process.env.LOGO || '/_nuxt-plenty/images/logo.svg',
+      favicon: process.env.FAVICON || '/_nuxt-plenty/favicon.ico',
+      ogTitle: process.env.OG_TITLE || 'PlentyONE Shop',
+      ogImg: process.env.OG_IMG || '/_nuxt-plenty/images/logo.svg',
+      manufactured: process.env.MANUFACTURED || '',
```

Comment:
These may not have any effect here. I think Favicon, OG Title and OG Image need to go into `app.config.ts` like in #1245